.dockerignore

```diff
@@ -5,4 +5,6 @@
 !package.json
 !package-lock.json
 !tsconfig.json
-!docker_start.sh
\ No newline at end of file
+!docker_start.sh
+!assets
```

Comment:
Do need assets anymore? as we are referring to cdnhost now

Reply:
This is required for local docker instance.